seed/static/seed/js/controllers/data_quality_admin_controller.js

```diff
@@ -222,8 +222,28 @@ angular.module('BE.seed.controller.data_quality_admin', [])
       var get_configured_rules = function () {
         var rules = [];
         var misconfigured_rules = [];
+        var duplicate_rule_group = [];
+        $scope.duplicate_rule_keys = [];
         _.forEach($scope.ruleGroups, function (ruleGroups, inventory_type) {
           _.forEach(ruleGroups, function (ruleGroup) {
+            var duplicate_rules = _.groupBy(ruleGroup, function(rule) {
+              return `${rule.condition}-${rule.field}-${rule.data_type}-${rule.min}-${rule.max}-${rule.text_match}-${rule.units}-${rule.severity}-${!_.isUndefined(rule.label)?rule.label:rule.status_label}`;
+            });
+            // Case 1: multiple keys with duplicate rules;
+            if (Object.keys(duplicate_rules).length > 1) {
+              _.forEach(duplicate_rules, function(group) {
+                if (group.length > 1) {
+                  _.forEach(group, function(rule) {
+                    $scope.duplicate_rule_keys.splice(0, 0, rule.$$hashKey);
+                  });
+                }
+              });
```

Comment:
I might be missing something, but why can't we use this in either of the two cases?